src/mixer.c

```diff
@@ -760,6 +760,54 @@ struct mixer_ctl *mixer_get_ctl_by_name_and_index(struct mixer *mixer,
     return NULL;
 }
 
+/** Gets an instance of mixer control handle, by the mixer control's name and device.
+ *  For instance, if two controls have same name,
+ *  e.g. 'Playback Channel map', then PCM device returns the specific control.
+ * @param mixer An initialized mixer handle.
+ * @param name The control's name in the given mixer.
+ * @param device The PCM device
+ * @returns A handle to the mixer control.
+ * @ingroup libtinyalsa-mixer
+ */
+struct mixer_ctl *mixer_get_ctl_by_name_and_device(struct mixer *mixer,
+                                                  const char *name,
```

Comment:
Please also help to add a space to line 773 and 774. Thanks.

Reply:
Sorry that I missed updating here. Updated now.